src/modal/modal.component.ts

```diff
@@ -152,6 +163,12 @@ export class Modal implements AfterViewInit, OnInit, OnDestroy {
 	 */
 	selectorPrimaryFocus = "[modal-primary-focus]";
 
+	/**
+	 * This is used within the component for automatic scrolling content detection. This can be overwritten
+	 * through public property `hasScrollingContent`.
+	 */
+	protected shouldShowScrollbar = false;
```

Comment:
This could be a `get`ter or a function so there's no need for `ngAfterContentChecked`, or `onResize`.

Reply:
Changing it to a getter and removing the`ngAfterContentChecked`, or `onResize` makes automatic detection on the alert modal not work